### ISSUE TITLE
CPU support for dilation rates larger than 1

### ISSUE THREAD

**Issue by bugie** [tensorflow/tensorflow#28264]
**System information**
- TensorFlow version (you are using): 1.13.1
- Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state.**
Current behavior for a model we are training is that CPU training yields errors:
`tensorflow/core/common_runtime/executor.cc:624] Executor failed to create kernel. Invalid argument: Current libxsmm and customized CPU implementations do not yet support dilation rates larger than 1.
         [[{{node Train_1/Optimizer/TrainOperation/gradients/Conv2D_72_grad/Conv2DBackpropFilter}}]]
`

GPU training is successful. We would like to have parity between CPU and GPU train as not all developers have a local GPU host. This appears to be a documented issue but I could not find any mention of when it may be fixed or what may be blocking this issue. 

https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/kernels/conv_grad_filter_ops.cc#L223

This is a feature request to complete the TODO mentioned in the above link. 

**Will this change the current api? How?**

**Who will benefit with this feature?**
Anyone that is trying to train on CPU for dilated convolutions
**Any Other info.**

**Comment by zaccharieramzi** [comment]
Hi, are there any updates on this?
In my use case, I don't necessarily want to train the model fully, but just test things out, which usually means I am using CPU.

**Comment by williamgilpin** [comment]
I am running into the same issue with both `Conv1D` and `Conv1DTranspose`. Thank you very much.

**Comment by timshen91** [comment]
@penpornk I'm told that you may work on CPU implementation? Can you take a look?

**Comment by ValueInvestorThijs** [comment]
This issue seems to persist. Has there been progress?

**Comment by SuryanarayanaY** [comment]
It's still unsupported as per master code.

https://github.com/tensorflow/tensorflow/blob/9dcf314237097784e02ff1979b10b6ddec9f26a8/tensorflow/core/kernels/conv_grad_filter_ops.cc#L160-L163

**Comment by Venkat6871** [comment]
Hi @bugie ,
Thank you so much for filing this feature request! We truly appreciate your support. At the moment, this request is not prioritized and mark it for community contributions. If you would like to contribute on a fix, please feel free to fork the repository and submit a pull request in accordance with the TensorFlow [CONTRIBUTION GUIDELINES](https://www.tensorflow.org/community/contribute/code) and link this issue in your PR to ensure the maintainers can easily review and collaborate.To avoid duplicate work, please leave a comment here if you decide to take it on!
Thank you again for your interest and we look forward to seeing your contributions!

**Comment by Venkat6871** [comment]
Hi @bugie ,
We are closing this issue due to inactivity. If this is still relevant and you would like to work on it, please feel free to reopen the issue or submit a pull request referencing it.

Thank you for your understanding.
